image-rs/tests/image_decryption.rs

```diff
@@ -30,7 +30,7 @@ const OCICRYPT_CONFIG: &str = "test_data/ocicrypt_keyprovider_ttrpc.conf";
 #[tokio::test]
 #[serial]
 async fn test_decrypt_layers(#[case] image: &str) {
-    common::prepare_test().await;
+    common::prepare_test("aa-offline_fs_kbc-resources.json").await;
```

Comment:
I think it would be better to declare this as a `const`, or we add the parameter to `#[case]` macro. How do you like?